GNUmakefile

```diff
@@ -91,6 +125,10 @@ upgrade-dcl:
 		sed ${SED_I} "s!.*declarative-resource-client-library.*!$$MOD_LINE!" go.mod.erb; echo "$$SUM_LINE" >> go.sum
 
 
+validate_environment:
```

Comment:
This seems similar to `mmv1/tools/doctor` - would it be possible to include the OUTPUT_PATH check there?

Reply:
doctor doesn't run as a part of initial build. It wouldn't make sense to include output path there as doctor is run to diagnose dependency issues not check build parameter settings.

Reply:
🤦 yes, that makes sense. However, the doctor might be possible to run instead of the check for goimports, since it already includes that (and a few other things)

Reply:
I'll take a look at running doctor.. I like the current solution since it's minimal, very fast and only outputs in case of failed detection. I'll take a look.

Reply:
updated